functions/count_char.py

```diff
@@ -14,19 +14,14 @@ def count_char(string,char):
     	string='hello selva',char='x'     | return -1
     	
     """	
-	if not string:
-		print "Invalid string"
-	elif not char:
-		print "No char is entered to find in string"
+	if char not in string:
+		return -1	
 	else:
```

Comment:
Don't use else part, since if char not in string your function will return, it wont go to next statement. if the character in string with out else itself other statement will get executed.